terminus-ui/tsconfig.json

```diff
@@ -4,21 +4,17 @@
     "target": "es2015",
     "rootDir": ".",
     "baseUrl": "",
+    "experimentalDecorators": true,
```

Comment:
Does this mean we won't have any more TypeScript errors in our .specs?!

Reply:
That's the hope.. I'm still getting mixed results in vim 😞